lib/Payment.js

```diff
@@ -775,6 +775,28 @@ class Payment {
       }
     );
   }
+
+  /**
+   * wechat international merchant only
+   * retrieve exchange rate for given foreign currency and date
+   * @param {object} query - visit https://pay.weixin.qq.com/wiki/doc/api/external/jsapi.php?chapter=9_15&index=10 for more info
+   * @return {Promise}
+   */
+  queryexchagerate(query) {
+    return this.simpleRequest(
+      this.paymentUrls.QUERY_EXCHANGE_RATE,
+      {
+        ...query,
```

Comment:
[object spread properties](https://node.green/#ES2018-features-object-rest-spread-properties-object-spread-properties) only supported on node-v8.3+, causing the CI failed